schema/geometry_schema.json

```diff
@@ -1,5 +1,6 @@
 {
 	"$schema": "http://json-schema.org/draft-04/schema#",
+	"version": "0.3.0",
```

Comment:
Let me add it to the schema, I have a fancy editor for handling json schema.  You are looking to add a version key at the top level, right above application and project?